app/views/pages/resource_library.html.haml

```diff
@@ -1,7 +1,7 @@
 - require 'csv'
```

Comment:
Should we also rename this file from `app/views/pages/resource_library.html.haml`